nix/sources.json

```diff
@@ -17,10 +17,10 @@
         "homepage": "https://github.com/NixOS/nixpkgs",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "8e1891d5b8d0b898db8890ddab73141f0cd3c2bc",
-        "sha256": "0a767mn0nfp4qnklsvs6bnc0vng4nc3ch566nmrz18ypk67z4zz0",
+        "rev": "d9896c1f8e419417ea8fc4b7e0e98ca4fc50ba8d",
+        "sha256": "1ysdab38jpjjvw5vc6qg1bns2f0hkwasww871hfhy9d9kxd6kaly",
```

Comment:
should the niv, nixpkgs bump also be mentioned in the release notes?

Reply:
It is only used in development dependencies. I will a note to the internal changes.